bin/fetch_xml_getty_records

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+# fetch.sh - download the most recent dump of xml files from the getty research portal
```

Comment:
Please update the docs to match the filename

Reply:
fixed in f206c8d50fa843531165af76fac8b53cbfccf1db